02-sight.Rmd

```diff
@@ -3,203 +3,70 @@ title: "Sightability"
-
-###### Missing values in the stratum or unit fields {-}
-
-These errors should be apparent when reviewing the data in the **Sampling** tab of the **Setup** page. Missing values in these columns can cause data to be discarded or cause the model to fail. To fix them navigate to the **Observation Entry** page, load the data for the survey in question, and fill in the missing information. Don't forget to click the button to save your changes when all the values have been entered.
-
-###### Empty rows at the bottom of the observation table {-}
-
-Sometimes when the metadata and observations entered for a survey don't agree there can be empty rows at the bottom of the observation data entry table. If that happens load the survey data in the **Observation Entry** page and right-click on the empty row(s) to delete. As above, don't forget to save your changes.
-
-## Step-By-Step Sightability Model Example {#sight-ex}
-
-This example shows how to run a sightability model for mule deer in the Smokey-Boise DAU step by step, including screenshots of the website.
-
-1. Start by clicking **Sightability** in the sidebar.
-
-![](./www/sight_01.PNG)
-
-2. Select the **Setup** page in the sidebar, then select the species and DAU you want to model. In this example we use mule deer and MD_Smokey-Boise. Make sure to select the correct species and DAU before clicking the button to load sightability data.
-
-3. Click the button labeled "Get Survey Data" to load the data for the species and DAU you selected.
-
-![](./www/sight_02.PNG)
-
-4. Once the data is loaded you should see a dialog indicating that the process was successful. Click Dismiss. If you receive an error see the [errors](#sight-errors) section.
-
-5. Review the data in the panes to the right (see [Loading Data](#sight-load) for details).
-
-![](./www/sight_03.PNG)
-
-6. Move on to the [**Options**](#sight-options) tab in the **User Inputs** pane, and make the appropriate selections. In this example we run a model on 2020 composition data for mule deer, with area as the spatial focus of the analysis.
-
-![](./www/sight_04.PNG)
-
-7. Move on to the [**Sampling**](#sight-options) tab and check that the data are accurate by selecting Sampling in the pane to the right. If the total population of subunits available in each stratum is incorrect you can fix it here.
-
-![](./www/sight_05.PNG)
+  <center>![](./www/sight_walk1.png)</center>
+  
+  Within the setup section, the User Inputs window decides which survey data you will load. Select your species, [<b>DAU</b>]{style="color: #086e04;"}, Survey Type, Year, Analysis Focus, and Aircraft/Model. The data preview on the right side of the page will update as you adjust your settings. Check it as you make your choices in the User Inputs window, but you do not have to modify it any further. 
```

Comment:
Should Species, Survey Type, Year, Analysis Focus, and Aircraft/Model also have definitions?